### ISSUE TITLE
Can we suppress the subfolders that pages are generated into?

### ISSUE THREAD

**Issue by fhaag** [TypeStrong/typedoc#2394]
## Search terms

subfolders

## Question

When Typedoc generates its article pages for each type, it sorts these files into subfolders according to what each of them describes - `enums`, `classes`, `interfaces`, and so on.

Can we suppress this, or tell Typedoc to use the same subfolder for all pages?

Background: Based on the module name and type name (which should be sufficiently unique; after all, we use nothing else when importing from another module in TypeScript), I am trying to automatically determine the URL to our Typedoc documentation. The fact that for this to work, I need to know whether a given identifier is e.g. a class, an interface, or anything else, is the only blocker left for me.

**Comment by Gerrit0** [comment]
Not yet supported - tracked in https://github.com/TypeStrong/typedoc/issues/2111

**Comment by fhaag** [comment]
@Gerrit0: I appreciate the link and I happily accept if it somehow covers my requirements.

However, I did find and read that other ticket before posting my question, and I just read it again, and both times, I saw little definitive overlap between my question and the other ticket. #2111 appears to be about a linking problem, restricted to functions, somehow specific to Cloudflare (?).

The ticket title ("Support for custom output directory structure") and a single sentence in the ticket ("I guess, what I'd like to see is a way to have customization over the file/folder structure.") sound reminiscent of what I'm asking about, but even the detailed description of the latter ("Maybe functions can be nested under their modules?") goes into a totally different direction.
